lib/theming/src/index.ts

```diff
@@ -1,23 +1,32 @@
 // eslint-disable-next-line @typescript-eslint/triple-slash-reference
 /// <reference path="./typings.d.ts" />
 
-import { CacheProvider } from '@emotion/core';
-import emotionStyled, { CreateStyled } from '@emotion/styled';
+import emotionStyled from '@emotion/styled';
+import * as emotionReact from '@emotion/react';
+// TODO: when releasing 7.0 this should be removed and code should be upgraded to use Emotion 11 types
+import type { CreateStyled, PropsOf, AddOptionalTo } from './emotion10types';
 
 import { Theme } from './types';
 
-export { CacheProvider };
```

Comment:
@Andarist I think this CacheProvider export is now removed, can we add this back?

Reply:
actually, never mind it IS exported on line 13!